doc/swish/http.tex

```diff
@@ -303,7 +303,12 @@ \subsection{WebSocket Protocol}\label{sec:websocket-protocol}
 
   \code{file-transform} & \code{(lambda (path) \#f)} & a procedure; a
   cache process calls this procedure with a file path and expects an
-  evaluator procedure or \code{\#f} (see \S\ref{sec:dynamic-pages})
+  evaluator procedure or \code{\#f} (see \S\ref{sec:dynamic-pages}) \\
+
+  \code{validate-path} & \code{http:valid-path?} & a predicate;
+  the server raises an exception if the \code{<request>} \var{path}
+  does not satisfy this prediate. \\
```

Comment:
prediate => predicate

Reply:
Oh dear, thanks for finding this. Fixed.